src/renderer/components/ft-list-video/ft-list-video.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <div
+    v-if="(!isLive && duration !== '0:00') || (!hideLiveStreams)"
```

Comment:
I am confused on this logic. Is this checking if the video is a live stream correct?
I may be completely wrong but does the if statement not trigger if: `(!isLive && duration !== '0:00')` is false?

Reply:
I should probably simplify the logic but it's basically 

```
if not a live stream or live streams aren't hidden then:
     show the video item
else
     display "Live Streams are hidden"

```

IIRC this code only matters for the Most Popular page
![image](https://user-images.githubusercontent.com/78101139/123552267-627be700-d743-11eb-98e1-1c3955a3bf10.png)

Reply:
My brain just can't get around it xD but if there are no logical issues in testing then that is fine.